zerver/openapi/zulip.yaml

```diff
@@ -9844,6 +9844,32 @@ paths:
                           **Changes**: Deprecated in Zulip 5.0 (feature level 89). Clients
                           connecting to newer servers should declare the `user_settings_object`
                           client capability and access the `user_settings` object instead.
+                      send_private_typing_notifications:
+                        type: boolean
+                        description: |
+                          Present if `update_display_settings` is present in `fetch_event_types`.
+
+                          Whether the user has chosen to send [typing notifications](/help/status-and-availability#typing-notifications)
```

Comment:
Should these be moved to `user_settings` object above? Or do we want to keep a copy here too with text like the following?
```
  **Changes**: Deprecated in Zulip 5.0 (feature level 89). Clients
  connecting to newer servers should declare the `user_settings_object`
  client capability and access the `user_settings` object.
````